node-operators/networks/run-a-node/systemd.md

```diff
@@ -334,6 +334,37 @@ For an overview of the flags used in the following start-up commands, plus addit
 
 ### Collator {: #collator }
 
+Beginning with v0.39.0, new Moonbeam collator nodes will no longer generate session keys automatically on start-up. When setting up a new node, run the following command to generate and store on disk the session keys that will be referenced in the start-up command:
```

Comment:
I would explicitly state that for existing nodes prior to v0.39.0, nothing needs to be done (same on docker page)